sips/sip-029/sip-029-halving-alignment.md

```diff
@@ -0,0 +1,159 @@
+# Preamble
+
+**SIP Number:** 029
+
+**Title:** Bootstrapping sBTC Liquidity and Incentive Preservation
+
+**Authors:**
+- Alex Miller (alex@hiro.so)
+- Andre Serrano (andre@bitcoinl2labs.com)
+- Brittany Laughlin (brittany@lattice.vc)
+- Jesse Wiley (jesse@stacks.org)
+- Jude Nelson (jude@stacks.org)
+- Philip De Smedt (philip@stackingdao.com)
+- Tycho Onnasch (tycho@zestprotocol.com)
+- Will Corcoran (will@stacks.org)
+
+**Consideration:** Economics, Technical, Governance
+
+**Type:** Consensus (hard fork)
+
+**Status:** Draft
+
+**Created:** 2024-11-06
+
+**License:** BSD 2-Clause
+
+**Sign-off:**
+
+**Discussions-To:**
+- Stacks Forum Post: [Aligning with Bitcoin Halving and Incentives after Nakamoto](https://forum.stacks.org/t/aligning-with-bitcoin-halving-and-incentives-after-nakamoto/17668)
+
+
+# Abstract
+
+The first Stacks halving is expected to take place at Stacks block height 210,384, which is set to occur during Reward Cycle 100 in December 2024, cutting the STX block reward from 1,000 STX to 500 STX. This SIP proposes a modification to the emissions schedule given that the network is going through two major launches (Nakamoto and sBTC) which rely on predictable economic incentives. The proposed schedule modification and associated STX emission rate would create time for Nakamoto and sBTC to launch and settle in, but, being mindful of supply, would still result in an overall reduced target 2050 STX supply (0.19% lower) and a reduced tail emission rate (20% lower).
+
+# License and Copyright
+
+This SIP is made available under the terms of the BSD-2-Clause license,
+available at https://opensource.org/licenses/BSD-2-Clause.  This SIP’s copyright
+is held by the Stacks Open Internet Foundation.
+
+# Introduction
+
+STX halvings were originally designed to happen every 4 years, similar to Bitcoin. The first Stacks halving is expected to take place at Stacks block height 210,384, which is set to occur during Reward Cycle 100 in December 2024, cutting the STX block reward from 1,000 STX to 500 STX, thereby significantly altering the economic incentives of mining on the Stacks blockchain. At the same time, the Stacks network is going through two major upgrades/changes, both of which are highly dependent on economic incentives.
```

Comment:
```suggestion
STX halvings were originally designed to happen every 4 years, similar to Bitcoin. The first Stacks halving is expected to take place at Stacks tenure height 210,384, which is set to occur during Reward Cycle 100 in December 2024, cutting the STX block reward from 1,000 STX to 500 STX, thereby significantly altering the economic incentives of mining on the Stacks blockchain. At the same time, the Stacks network is going through two major upgrades/changes, both of which are highly dependent on economic incentives.
```

Reply:
This text is wrong.  The first Stacks halvening was targeted to take place at Bitcoin height 876,430.  Stacks block heights (or tenure heights) have nothing to do with it.

Reply:
good catch, i missed that on my readthrough. @will-corcoran jude is correct here and the text should reference the bitcoin block, not the stacks block (especially post 3.0, we can't be certain which stacks block it will be).

Reply:
unless my calculation is off, it looks like btc block 876434

https://github.com/stacks-network/stacks-core/blob/master/stackslib/src/chainstate/stacks/db/blocks.rs#L3638

which should arrive in late december based on current block times.

Reply:
Yup, I arrived at the same conclusion above before seeing your comment here.

Reply:
Noted.  I will correct this now.  Thanks @jcnelson @wileyj @obycode